service/lint.ml

```diff
@@ -0,0 +1,19 @@
+open Current.Syntax
+module Docker = Current_docker.Default
+
+let format_dockerfile ~base ~ocamlformat_version =
+  let open Dockerfile in
+  from (Docker.Image.hash base)
+  @@ run "git -C /home/opam/opam-repository pull"
```

Comment:
This will prevent most caching of the installation of ocamlformat, since every build will get a slightly different version of the repository. Is it really needed?

Reply:
Indeed; it's not really needed.